internal/model/model.go

```diff
@@ -1653,6 +1653,14 @@ func (m *Model) BringToFront(folder, file string) {
 // CheckFolderHealth checks the folder for common errors and returns the
 // current folder error, or nil if the folder is healthy.
 func (m *Model) CheckFolderHealth(id string) error {
+	// This could be expensive wrt sleeping disks... if random(1) < 0.2 ?
+	usage, syserr := osutil.DiskUsedPercentage(m.cfg.ConfigPath())
+	if syserr == nil && usage > 99 {
```

Comment:
1. I think we should panic here, the db could become corrupted at any time now. It's just that the panic gets recovered somehow. But if you don't like the panic I won't investigate it
2. What about wrapping it with ```if rand.Float64() < 0.2``` to keep things cheap? Should I add it?